swri_transform_util/test/initialize_origin_manual.test.py

```diff
@@ -63,13 +63,25 @@ def generate_test_description():
         parameters=[{
             "local_xy_frame": "/far_field",
             "local_xy_origin": "swri",
-            "local_xy_origins": [29.45196669, -98.61370577, 233.719, 0.0],
+            "local_xy_origins": [29.45196669, -98.61370577, 233.719],
+        }]
+    )
+
+    init_origin2 = Node(
+        package="swri_transform_util",
+        name="origin",
+        executable="initialize_origin.py",
+        parameters=[{
+            "local_xy_frame": "/far_field",
+            "local_xy_origin": "swri",
+            "local_xy_origins": "[{name: 'swri', latitude: 29.45196669, longitude: -98.61370577, altitude: 233.719}]",
         }]
     )
 
     return launch.LaunchDescription(
             [
                 init_origin,
+                init_origin2,
```

Comment:
I'd prefer a more descriptive name here.

Reply:
👍

Reply:
Updated the names of both to be a bit more descriptive.